WultraMobileTokenSDK/Operations/Service/WMTOperationsImpl.swift

```diff
@@ -131,6 +131,7 @@ class WMTOperationsImpl<T: WMTUserOperation>: WMTOperations, WMTService {
     private var isPollingPaused: Bool { return pollingTimer?.isValid == false }
     private let pollingLock = WMTLock()
     private var notificationObservers = [NSObjectProtocol]()
+    private let minimumTimePollingInterval = TimeInterval(5)
```

Comment:
TimeInteval is just an alias for a `Double` type, The `private let minimumTimePollingInterval: TimeInterval = 5` or just `private let minimumTimePollingInterval = 5.0` is enough, `Double(5)` is not used as well.